src/components/StakingHierarchy.js

```diff
@@ -0,0 +1,465 @@
+            or feel comfortable staking 32 ETH.
+          </p>
+          <p>
+            Many of these options include what is known as "liquid staking"
+            which involves an ERC-20 liquidity token that represents your staked
+            ETH.
+          </p>
+          <p>
+            This enables easy and any-time exiting, and makes staking as simple
+            as a token swap. These options also allow/require users to hold
+            custody of their assets in their own Ethereum wallet.
+          </p>
+          <p>
+            Pooled staking is not native to the Ethereum network, and thus have
+            been built out by third parties, and carry their own risk.
+          </p>
+          <ButtonLink
+            onClick={() => {
+              trackCustomEvent({
+                eventCategory: `StakingHierarchy`,
+                eventAction: `Clicked`,
+                eventName: "clicked pooled staking",
+              })
+            }}
+            to="/staking/pools/"
+          >
+            More on pooled staking
+          </ButtonLink>
+        </Content>
+      </Section>
+      <Section number="4">
+        <Ether>
+          <StyledEtherSvg size="70%" />
+        </Ether>
+        <Line />
+        <Header>
+          <h2>Centralized exchanges</h2>
+          <Pills className="subtext">
+            <p>
+              <em>Least impactful</em>
+            </p>
+            <p>Highest trust assumptions</p>
+          </Pills>
+        </Header>
+        <Glyph>
+          <CexGlyph />
+        </Glyph>
+        <Content>
+          <p>
+            Ethereum users are encouraged to stake by the best means possible
+            for the health and decentralization of the network, which tends to
+            involve individuals taking control instead of centralized agencies.
+          </p>
+          <p>
+            If you are not yet comfortable holding ETH in your own wallet, many
+            centralized exchanges provide staking services and can be a fallback
+            to allow you to earn some yield on your ETH holdings, with minimal
+            oversight or effort.
+          </p>
+          <p>
+            The trade-off here is that centralized providers consolidate large
+            pools of ETH to run large numbers of validators. This can be
+            dangerous for the network and its users as it creates a large
+            centralized target and point of failure, making the network more
+            vulnerable to attack or bugs.
+          </p>
+          <p>
+            Users should stake by the best means possible,{" "}
+            <em>within their means</em>. If you don't feel comfortable holding
+            your own keys, that's okay. These options are here for you.
+          </p>
+          <p>
+            In the meantime, consider checking out our{" "}
+            <Link to="/wallets">wallets page</Link>, where you can get started
```

Comment:
```suggestion
            <Link to="/wallets/">wallets page</Link>, where you can get started
```
Note we should always use trailing slashes for links.